Messaging.md

```diff
@@ -87,33 +97,126 @@ The following DLC-specific types are used throughout the specification. All type
 
 ### The `contract_info` Type
 
-This type contains information about a contracts outcomes and their corresponding payouts. To save space, only one side's POV is included in this message as the other can be derived using `remote_payout = total_collateral - local_payout`.
+This type contains information about a contract's outcomes, their corresponding payouts, and the oracles to be used.
 
 #### Version 0 `contract_info`
 
-1. type: 42768 (`contract_info_v0`)
+1. type: 55342 (`contract_info_v0`)
 2. data:
-   * [`bigsize`:`nb_outcomes`]
-   * [`sha256`:`outcome_1`]
-   * [`u64`:`outcome_1_local_payout`]
+   * [`u64`:`total_collateral`]
+   * [`contract_descriptor`:`contract_descriptor`]
+   * [`oracle_info`:`oracle_info`]
+
+`total_collateral` is the Satoshi-denominated value of the sum of all party's collateral.
+
+### The `contract_descriptor` Type
+
+This type contains information about a contract's outcomes and their corresponding payouts.
+
+To save space, only the offerer's POV is included in this message as the other can be derived using `accept_payout = total_collateral - offer_payout`.
```

Comment:
nit: I guess POV means Payout Value? It's not very clear to me, and it's never defined so might be better to just spell it out.

Reply:
Funny enough both point of view and payout value work here lol, will fix tho